obspy/signal/spectral_estimation.py

```diff
@@ -2020,7 +2053,48 @@ def __plot_coverage(self, ax):
 
         ax.autoscale_view()
 
-
+def get_earthquakes():
```

Comment:
Since this is static, I'd prefer this to be a constant (e.g. `EARTHQUAKE_MODELS = {...}`) at top of file. You'll also need spaces after colons and commas.

Reply:
OK.